test/recipes/70-test_tls13messages.t

```diff
@@ -134,6 +134,11 @@ $ENV{CTLOG_FILE} = srctop_file("test", "ct", "log_list.conf");
     [0,0,0]
 );
 
+use constant {
+    PURE_SIGALGS => 0,
+    COMPAT_SIGALGS => 1
+};
+
```

Comment:
Sorry, I didn't spot this when I looked at this before. These tests shouldn't really be in this file. The tests in this file are all about sanity checking that we see all the right handshake messages in the right order with the right extensions. Your added tests are about testing the sanity of the contents of the sigalgs extension, which is better placed in 70-test_sslsigalgs.t